src/Rules/Drupal/EntityQuery/EntityQueryHasAccessCheckRule.php

```diff
@@ -40,7 +40,7 @@ public function processNode(Node $node, Scope $scope): array
 
         return [
             RuleErrorBuilder::message(
-                'Missing explicit access check on entity query.'
+                'Relying on entity queries to check access by default is deprecated in drupal:9.2.0 and an error will be thrown from drupal:10.0.0. Call \Drupal\Core\Entity\Query\QueryInterface::accessCheck() with TRUE or FALSE to specify whether access should be checked. See https://www.drupal.org/node/3201242.'
```

Comment:
We need to remove `See https://www.drupal.org/node/3201242`, it's added as a tip in the PHPStan error output